TotalCrossVM/docker/arm32v7/build-docker.sh

```diff
@@ -4,4 +4,4 @@ export DOCKER_BUILDKIT=1
 docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
 docker buildx create --use --name armbuilder
 docker buildx inspect --bootstrap
-docker buildx build --platform linux/arm/v7 --load -t totalcross/linux-arm32v7-build .
+docker buildx build --no-cache --platform linux/arm/v7 --load -t totalcross/linux-bionic-arm32v7-build .
```

Comment:
```suggestion
docker buildx build --platform linux/arm/v7 --load -t totalcross/linux-bionic-arm32v7-build .
```
commited the --no-cache by mistake